go.mod

```diff
@@ -1,23 +1,21 @@
 module github.com/go-jet/jet/v2
 
-go 1.18
+go 1.22.0
 
 require (
 	github.com/go-sql-driver/mysql v1.8.1
+	github.com/google/go-cmp v0.6.0
 	github.com/google/uuid v1.6.0
 	github.com/jackc/pgconn v1.14.3
-	github.com/lib/pq v1.10.9
-	github.com/mattn/go-sqlite3 v1.14.17
-)
-
-require (
```

Comment:
The idea between the two groups was that one is for the library dependencies and the second is for test dependencies.

Reply:
Rolled back. Does go differentiate between the two?  Unless it's just for readability? My understanding that the dependencies included are based on the imports actually invoked in the dependency graph.  ie. 2 different main.go would have 2 different dependency trees.

Reply:
Just for readability.